python/strings.cpp

```diff
@@ -21,23 +21,10 @@ std::string context_string(const arb::context& c) {
     const bool mpi = arb::has_mpi(c);
     s << "<context: threads " << arb::num_threads(c)
       << ", gpu " << (gpu? "yes": "None")
-      << ", distributed " << (mpi? "MPI": "Local")
+      << ", distributed " << (mpi? "MPI": "local")
       << " ranks " << arb::num_ranks(c)
       << ">";
     return s.str();
 }
 
-std::string proc_allocation_string(const arb::proc_allocation& a) {
```

Comment:
Regarding converting types to strings, we could consider using a string formatting library, like fmt (see https://github.com/fmtlib/fmt), or use our own `pprintf` implementation that is in `arbor/util/strprintf.hpp`.

The benefit of such a function is that it would make it easy to write lambdas that generate strings, doing away with the need for stand alone functions. For now, don't worry about it.

Reply:
Lleft open for now.